webapp/src/components/surveyDateSelector/surveyDateSelector.tsx

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React, {useCallback} from 'react';
+import {format} from 'date-fns';
+
+import DatePicker from 'components/common/datePicker';
+
+import './style.scss';
+
+export type Props = {
+    value: Date
+    onChange?: (value: Date) => void
+};
+
+const SurveyDateSelector = ({value, onChange}: Props) => {
+    const onSelectHandler = useCallback((newValue: Date) => {
+        if (onChange) {
+            onChange(newValue);
+        }
+    }, [onChange]);
+
+    return (
+        <DatePicker
+            onSelect={onSelectHandler}
+            value={value}
+        >
+            <div className='SurveyDateSelector form-control'>
```

Comment:
Not sure if it's something you can or want to change, but `form-control` is actually a CSS class from Bootstrap that's leaking into here. I'd personally try to get rid of it, but if we need it for consistent styling, then it's totally fine to leave it

Reply:
yeah it's for styling. It applies a loooooot of styling that was taking too long to apply manually so I just used the class name.